BurnOutSharp/ProtectionType/IndyVCD.cs

```diff
@@ -13,8 +13,8 @@ public ConcurrentQueue<string> CheckDirectoryPath(string path, IEnumerable<strin
             // TODO: Verify if these are OR or AND
             var matchers = new List<PathMatchSet>
             {
-                new PathMatchSet(new PathMatch("INDYVCD.AX", useEndsWith: true), "IndyVCD"),
-                new PathMatchSet(new PathMatch("INDYMP3.idt", useEndsWith: true), "IndyVCD"),
+                new PathMatchSet(new PathMatch("INDYVCD.AX", useEndsWith: true), "IndyVCD (Unconfirmed - Please report to us on Github)"),
```

Comment:
Please double check these because I think these were already confirmed from a screenshot or something.

Reply:
There was a forum post online, but not a direct screenshot.